tooling/maven-plugin/src/main/java/org/apache/camel/quarkus/maven/CqCatalog.java

```diff
@@ -111,6 +111,10 @@ public static CqCatalog findFirstFromClassPath() {
     public CqCatalog(Path baseDir, Flavor flavor) {
         this(flavor, name -> {
             try {
+                //workaround for beans
+                if (name.contains("org/apache/camel/catalog/beans/")) {
+                    return Thread.currentThread().getContextClassLoader().getResourceAsStream(name);
+                }
```

Comment:
IIUC, this workaround needs to be removed when the upstream camel  bean typo issue fixed.

Reply:
I'll replace this workaround  with custom catalog workaround, which is better.